docs/source/getting-started/setting-up-driver-plugin.adoc

```diff
@@ -77,6 +77,33 @@ $ sudo usermod -a -G libvirt <username>
 $ newgrp libvirt
 ----
 
+[[kvm-driver-arch-linux]]
+=== On Arch Linux
```

Comment:
What's the specific part? And why do we want to document it explicitly?

Reply:
I am using Mint? Can you add instructions for that as well?

Reply:
we got a lot of questions on the IRC about it also an issue got opened, the specific parts are the two seperate packages that has to be installed and making changes to the `/etc/libvirt/qemu.conf`.

Reply:
Sure, if the majority of @minishift/minishift-dev agrees, let's add it. I'm neutral

Reply:
I am in favor of adding it as it involves an extra step. I am going to approve this PR. However we should just add the extra step required for Arch Linux and we should not repeat steps. May be we can take another look later to organize this better.

Reply:
Linux Mint and Ubuntu are identical in setup, but Arch Linux involves additional steps. This is particularly an important is distro in Asia which is used by our target audience